web/src/utils/storybook.ts

```diff
@@ -194,7 +196,7 @@ const getFutureValues = () => {
           datetime,
           temperature: temp + (Math.random() - 0.5) * 8,
           bias_adjusted_temperature: temp + (Math.random() - 0.5) * 8 - 2,
-          dew_point,
+          dewpoint,
```

Comment:
Do we not also want the random variation here?

Reply:
This is in the section for GDPS values, which is unrelated to this ticket. I just renamed the variable for consistency here, but it would probably be less confusing if I just deleted it altogether. At this point we're not using dewpoint for GDPS at all, so it's not necessary